packages/node/src/plugins/webpackChunkUtilities.ts

```diff
@@ -280,42 +280,10 @@ export function generateLoadScript(runtimeTemplate: any): string {
             if (!name) {
               throw new Error('__webpack_require__.l name is required for ' + url);
             }
-            var remoteName = name;
-            if(name.includes('__remote_scope__')) {
-              remoteName = name.split('__remote_scope__.')[1]
```

Comment:
@2heal1 see here, i used to split name to see if its scoped on global. 
I think it would be a good idea to use this still but in a new way as mentioned in other comment. This makes it easy to clean up global objet if we need to HMR, we can just clear moduleCache or something like i used to do to __remote_scope__ = {}; to reset